tasks/release.py

```diff
@@ -0,0 +1,99 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from __future__ import print_function, unicode_literals
+import os
+
+from packaging import version
+from invoke import task, call
+from invoke.exceptions import Exit
+
+from .constants import AGENT_BASED_INTEGRATIONS, ROOT
+from .utils import get_version_string, get_current_branch, get_release_tag_string
+from .changelog import do_update_changelog
+
+
+def update_version_module(check_name, old_ver, new_ver):
+    """
+    Change the Python code in the __about__.py module so that `__version__`
+    contains the new value.
+    """
+    about_module = os.path.join(check_name, 'datadog_checks', check_name, '__about__.py')
+    with open(about_module, 'r') as f:
+        contents = f.read()
+
+    contents = contents.replace(old_ver, new_ver)
+    with open(about_module, 'w') as f:
+        f.write(contents)
+
+
+def git_commit(ctx, target, message):
+    """
+    Commit the current changes.
+    """
+    target_path = os.path.join(ROOT, target)
+    cmd = 'git add ' + target_path
+    ctx.run(cmd)
+    cmd = 'git commit -m"{}"'.format(message)
+    ctx.run(cmd)
+
+
+def git_tag(ctx, tag_name):
+    """
+    Tag the repo using an annotated tag.
+    """
+    cmd = 'git tag {} -m"{}"'.format(tag_name, tag_name)
```

Comment:
Do we need a `-a`?

Reply:
from the manual:
> If -m <msg> or -F <file> is given and -a, -s, and -u <keyid> are absent, -a is implied.
maybe it's better to explicitly add `-a`

Reply:
I didn't know that, thanks!